### PR TITLE
swamp/tests: light node syncs with multiple trusted peers

### LINKED ISSUE #418: swamp/tests/sync: Light node can be initialised with multiple trusted peers
ATM, the light node can be initialised with only 1 trusted peers. Soon this will be changed due to https://github.com/celestiaorg/celestia-node/issues/364

We need a sync test that checks that this is possible and a negative case where one of the trusted peer is down, but still the light node continues with syncing process

### PR DESCRIPTION
**Pull request by Bidon15**
- Edit WithTrustedPeers to accept a slice of strings/multiaddresses
- Adapt flags
- Minor prettifying of how we treat addr[0] as a separate var called `trustedPeers`
- Increased timeout context for start/stop for stable local execution on macOS, when user executes `make test` in cli

Resolves #418

### CODE REVIEW

File: node/config_opts.go [hunk 1]
```diff
@@ -18,10 +18,10 @@ func WithTrustedHash(hash string) Option {
 	}
 }
 
-// WithTrustedPeer appends TrustedPeer to the Config.
-func WithTrustedPeer(addr string) Option {
+// WithTrustedPeers appends a slice of TrustedPeers to the Config.
+func WithTrustedPeers(addr []string) Option {
```
Comment (Wondertan) [inline node/config_opts.go:22]:
```suggestion
// WithTrustedPeers appends new "trusted peers" to the Config.
func WithTrustedPeers(addr ...string) Option {
```

Comment (Wondertan) [reply node/config_opts.go:22]: This way you won't need to wrap values with an array and simply pass them in options as many as you want

File: node/tests/sync_test.go [hunk 2]
```diff
@@ -217,3 +223,66 @@ func TestSyncLightWithFull(t *testing.T) {
...
+2. Start a BN
+3. Check BN is synced to height 20
+4. Create a Full Node(FN) with a connection to BN as a trusted peer
+5. Start a FN
+6. Check FN is synced to height 30
+7. Create a Light Node(LN) with a connection to BN, FN as trusted peers
+8. Start LN
+9. Check LN is synced to height 50
+*/
+func TestSyncLightWithMultiplePeers(t *testing.T) {
```
Comment (Wondertan) [inline node/tests/sync_test.go:243]:
```suggestion
func TestSyncLightWithTrustedPeers(t *testing.T) {
```

**Comment by Wondertan** [comment]
Pls add the kind label

File: node/tests/sync_test.go (hunk 2 above)
Comment (Bidon15) [reply node/tests/sync_test.go]: Done

File: node/config_opts.go (hunk 1 above)
Comment (Bidon15) [reply node/config_opts.go]: Done

**Review by Wondertan — APPROVED** [review]

**Review by vgonkivs — APPROVED** [review]
